templates/flutter/lib/src/client_browser.dart.twig

```diff
@@ -1,4 +1,7 @@
 import 'dart:html' as html;
+import 'dart:io';
```

Comment:
As this is browser_client, we should not use dart:io, cookie_jar, here, we should handle this some other way